### PR TITLE
Add classname and line number to azkaban logs

### PR DESCRIPTION
**Pull request by pcadabam**
@liyinan926 and @chavdar can you guys take a looks this?
- I spoke to the Azkaban team today as to why the azkaban logs do not have class names and log messages even though we have it in our log4j conifg files. It turns out the azkaban hard codes the pattern to just project name and message. In fact it seems like azkaban ignores all the log4j properties we set.
- This change is to add the class name and line number to all the appenders in the root logger.

### CODE REVIEW

File: gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanJobLauncher.java [hunk 1]
```diff
@@ -126,6 +132,22 @@ public AzkabanJobLauncher(String jobId, Properties props)
...
+   */
+  private void updateLogger() {
+
+    @SuppressWarnings("unchecked")
+    Enumeration<Appender> appenders = Logger.getRootLogger().getAllAppenders();
+
+    if (appenders != null) {
+      while (appenders.hasMoreElements()) {
+        appenders.nextElement().setLayout(new PatternLayout(LOGGER_PATTERN_LAYOUT));
+
```
Comment (liyinan926) [inline gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanJobLauncher.java]: Additional empty line.

File: gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanJobLauncher.java [hunk 2]
```diff
@@ -126,6 +132,22 @@ public AzkabanJobLauncher(String jobId, Properties props)
     return tags;
   }
 
+  /**
+   * Azkaban ignores the log4j config provided in the artifacts. This method includes class name and line number to azkaban logs
```
Comment (liyinan926) [inline gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanJobLauncher.java]: Can you wrap the Javadoc and add dot at the end?

**Comment by liyinan926** [comment]
LGTM other than the minor comments.

**Comment by chavdar** [comment]
LGTM

**Pull request closed without merging**